src/App.js

```diff
@@ -1,7 +1,65 @@
 import React from 'react';
+import { Routes, Route, Link } from 'react-router-dom';
 import './App.css';
 import Calculator from './components/Calculator';
 
+function NavBar() {
```

Comment:
- [ ] Please extract this into a single component to make your app maintainable, you can also reuse this component somewhere else and it is always best practice to use an arrow function